v3/plugins/gattytto/Dart-Code/3.15.0/meta.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+publisher: gattytto 
+name: Dart-Code
+version: 3.15.0
+type: VS Code extension
+displayName: Dart
+title: Google's Dart language support
+description: This extension adds rich language support for the Dart syntax
+icon: https://dart.dev/assets/shared/dart/logo+text/horizontal/white-e71fb382ad5229792cc704b3ee7a88f8013e986d6e34f0956d89c453b454d0a5.svg
+repository: https://github.com/gattytto/Dart-Code
```

Comment:
hi,

why are we using your fork and not https://github.com/Dart-Code/Dart-Code ?

Reply:
because I needed to make changes to the extension code so it doesn't break on che, and this requires to fork it so it passes all the checks

Reply:
[this](https://github.com/eclipse/che/issues/17732#issuecomment-682309957) is the relevant comment for  the issue, as long as another change I did which is mentioned in the extension vsix name "realpathSync"

Reply:
what I did is to add the relevant changes to the current master branch of the official extension so we get both fixes and latest extension changes so far. which results in a kind of 3.15.0 version with commented parts of the code so it doesn't break.

Reply:
hi, we can't have these changes merged upstream ?

Reply:
in https://github.com/Dart-Code/Dart-Code

because with the forks, releases will be out-of sync and it's adding extra maintenance

Reply:
We've been planning to support multi-root workspaces for a _long_ time. How about we prioritize that task once again?

Reply:
> in https://github.com/Dart-Code/Dart-Code
> because with the forks, releases will be out-of sync and it's adding extra maintenance

We can go back to official releases after the theia issue has been fixed -- it's not clear if it will be fixed immediately